### DISCUSSION TITLE
'Fleet Name' form field unification

### DISCUSSION THREAD

**Discussion by Gadicuz** [ppfeufer/alliance-auth-app-translations#2] [General]
### Bug Description

I guess some style unification required for 'Fleet Name' field among AA applications.

### Steps to Reproduce

***** AA SRP *****
```
srp_name = forms.CharField(
  required=True,
  label=get_mandatory_form_label_text(_("Fleet Name"))
)
```
***** AA Fleetpings *****
```
fleet_name = forms.CharField(
  required=False,
  label=_("Fleet Name"),
  max_length=254,
  widget=forms.TextInput(attrs={"placeholder": _("What is the fleet name in fleet finder?")}),
)
```
***** Allianceauth AFAT *****
```
name_esi = forms.CharField(
  required=True,
  label=get_mandatory_form_label_text(_("Fleet Name")),
  max_length=255,
  widget=forms.TextInput(attrs={"placeholder": _("Enter fleet name")}),
)
```
***** Allianceauth AFAT *****
```
name = forms.CharField(
  required=True,
  label=get_mandatory_form_label_text(_("Fleet Name")),
  max_length=255,
  widget=forms.TextInput(attrs={"placeholder": _("Enter fleet name")}),
)
```
***** Allianceauth AFAT *****
```
fleet = forms.CharField(
  required=True,
  label=get_mandatory_form_label_text(_("Fleet Name")),
  max_length=255,
)
```

The fields have different max_length and placeholder text. Some doesn't have any.

### Expected behavior

Same user experience for 'Fleet Name' form field among all applications and within any application. Text label strings could be added to the translation glossary.

**Comment by ppfeufer** [comment]
Hey,

I transferred this issue to its own repository to collect all issues with translations, since this is not just related to AA-SRP.

You got a good point about adding consistency!
